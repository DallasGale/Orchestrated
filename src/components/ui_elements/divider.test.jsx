import React from 'react'
import { render } from 'enzyme'
import toJson from 'enzyme-to-json'
import Divider from './divider'

describe('<Divider />', () => {
  test('it renders', () => {
    const Component = render(<Divider />)
    expect(toJson(Component)).toMatchSnapshot()
  })
})
