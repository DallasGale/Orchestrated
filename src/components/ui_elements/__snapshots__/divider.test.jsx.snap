// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Divider /> it renders 1`] = `
<div
  class="sc-bdVaJa hnIxNj"
/>
`;
